lib/src/drm/playready/getLicenseRequestFromMessage.ts

```diff
@@ -0,0 +1,68 @@
+import { base64decode } from '../../utils/base64decode';
+import { parseXml, type XmlNode } from '../../xml';
+import { CHALLENGE } from '../common/CHALLENGE';
+import type { ENCODING_UTF8 } from '../common/ENCODING_UTF8';
+import { ENCODING_UTF16 } from '../common/ENCODING_UTF16';
+import { LICENSE_ACQUISITION } from '../common/LICENSE_ACQUISITION';
+import { PLAYREADY_KEY_MESSAGE } from '../common/PLAYREADY_KEY_MESSAGE';
+
+/**
+ * Gets the PlayReady license request from the MediaKeyMessageEvent.
+ *
+ * @param message - An ArrayBuffer from the content decryption module.
+ * @param encoding - The message encoding type. Default is UTF-16.
+ * @returns The license request as an ArrayBuffer.
+ *
+ * @group DRM
+ * @beta
+ *
+ */
+export function getLicenseRequestFromMessage(
+	message: ArrayBuffer,
+	encoding: typeof ENCODING_UTF8 | typeof ENCODING_UTF16 = ENCODING_UTF16,
```

Comment:
@willdharris We also need to add all the new DRM related functions and constants we are exporting to the list of exports in `drm.ts`.